app/dashboards/visitor_dashboard.rb

```diff
@@ -61,7 +61,7 @@ class VisitorDashboard < Administrate::BaseDashboard
   # Overwrite this method to customize how visitors are displayed
   # across all pages of the admin dashboard.
   #
-  # def display_resource(visitor)
-  #   "Visitor ##{visitor.id}"
-  # end
+  def display_resource(visitor)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)